Makefile

```diff
@@ -704,9 +705,13 @@ check-symbols: startup_files libc
 	    | grep -v '^#define __GCC_HAVE_SYNC_COMPARE_AND_SWAP_\(1\|2\|4\|8\)' \
 	    > "$(SYSROOT_SHARE)/predefined-macros.txt"
 
-	# Check that the computed metadata matches the expected metadata.
-	# This ignores whitespace because on Windows the output has CRLF line endings.
-	diff -wur "$(CURDIR)/expected/$(TARGET_TRIPLE)" "$(SYSROOT_SHARE)"
+	# Only verify metadata for default set of extra cflags; otherwise, it
+	# most likely won't match.
+ifeq ($(EXTRA_CFLAGS),$(DEFAULT_EXTRA_CFLAGS))
+		# Check that the computed metadata matches the expected metadata.
+		# This ignores whitespace because on Windows the output has CRLF line endings.
+		diff -wur "$(CURDIR)/expected/$(TARGET_TRIPLE)" "$(SYSROOT_SHARE)"
```

Comment:
Does this extra indentation work?   Personally, I would rather not indent this an extra level like this.

Reply:
Yes, this works. I was hoping that I can indent `ifeq` but apparently it has to start from the beginning of the line...

I don't really have a preference for that; either way looks equally ugly to me so if you prefer shifting lines L711..713 left by one tab, I can do that.